tests/integration/dynamic_ip_test.py

```diff
@@ -42,6 +43,7 @@
 from .testlib.bridgelib import create_bridge_subtree_state
 from .testlib.bridgelib import linux_bridge
 
+DEFAULT_TMO = 20
```

Comment:
Please call this `DEFAULT_TIMEOUT` or ideally something that scopes the timeout, not sure what kind of timeout it actually is but it should be more clear from its name. It is a timeout, isn't it?

Reply:
Sorry. I through people expect TMO == TIMEOUT.
Anyway, changed.